openapi.yaml

```diff
@@ -177,7 +136,7 @@ paths:
           
     post:
       tags:
-        - users
+        - featureFlags-User
       summary: To enable feature flag for a user
```

Comment:
how do we disable a FF for a user ?

Reply:
If we want to check when it was deactivated we may require `deactivatedAt`  attribute of type `timestamp`